media/media.css

```diff
@@ -59,13 +59,55 @@ body {
 
 .container-infos {
   display: flex;
-  flex-wrap: wrap;
   height: calc(100vh - 15rem);
   overflow: scroll;
 }
 
+.templates {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  height: calc(100vh - 15rem);
+  overflow: scroll;
+}
+
+.templates div {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.piral-template, .pilet-template {
+  width: 13.75rem;
+  max-width: 13.75rem;
+  margin: 0 1rem 1rem 0;
+  text-align: center;
+  min-width: 9rem;
```

Comment:
I don't get `min-width` and `max-width` when `width` is set to a non-relative value.